src/main/webapp/WEB-INF/views/ContactSupport.html

```diff
@@ -0,0 +1,203 @@
+                <li class="active"><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Projects</a></li>
+                <li>
+                    <form role="search" class="navbar-form navbar-left">
+                        <div class="form-group">
+                            <input type="text" placeholder="Search" class="form-control">
+                        </div>
+                    </form>
+                </li>
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
+                <li class="active"><a href="#modal-login" data-toggle="modal">Login</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+<div class="container">
+    <div id="contacts">
+        <div class="row">
+            <div class="col-sm-offset-3 col-sm-6">
+                <form name="sentMessage" class="well" id="contactForm"  novalidate>
+                    <legend>Contact Support</legend>
+                    <div class="control-group">
+                        <div class="controls">
+                            <input type="text" class="form-control"
+                                   placeholder="Full Name" id="name" required
+                                   data-validation-required-message="Please enter your name" />
+                            <p class="help-block"></p>
+                        </div>
+                    </div>
+                    <div class="control-group">
+                        <div class="controls">
+                            <input type="email" class="form-control" placeholder="Email" id="email" required
+                                   data-validation-required-message="Please enter your email" />
+                        </div>
+                    </div>
+
+                    <div class="control-group">
+                        <div class="controls">
+						<textarea rows="10" cols="100" class="form-control"
+                                  placeholder="Message" id="message" required
+                                  data-validation-required-message="Please enter your message" minlength="5"
+                                  data-validation-minlength-message="Min 5 characters"
+                                  maxlength="999" style="resize:none"></textarea>
+                        </div>
+                    </div>
+                    <div id="success"> </div>
+                    <button type="submit" class="btn btn-primary pull-right">Send</button><br />
+                </form>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="modal-login">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">Registration
+                    <button class="close" type="button" data-dismiss="modal">
+                        <i class="fa fa-close"></i>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)